dev/sh/skaffold.yaml

```diff
@@ -36,36 +35,26 @@ deploy:
       - https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/dev/sh/gcloud-sqlproxy.yaml
     # datarepo-api
     - name: sh-jade-datarepo-api
-      chartPath: https://github.com/broadinstitute/datarepo-helm/releases/download/datarepo-api-0.0.25/datarepo-api-0.0.25.tgz
-      version: 0.0.25
+      chartPath: https://github.com/broadinstitute/datarepo-helm/releases/download/datarepo-api-0.0.23/datarepo-api-0.0.23.tgz
+      version: 0.0.83
       namespace: sh
       remote: true
       values:
         imageName: gcr.io/broad-jade-dev/jade-data-repo
       valuesFiles:
-      - https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/dev/sh/datarepo-api.yaml
-    #Point to local datarepo-helm api chart
-    # datarepo-api local helm chart
-      #- name: sh-jade-datarepo-api
-      #chartPath: ../datarepo-helm/charts/datarepo-api/
-      #remote: false
-      #namespace: sh
-      #values:
-      #  imageName: gcr.io/broad-jade-dev/jade-data-repo
-      #valuesFiles:
-      #  - https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/dev/sh/datarepo-api.yaml
+      - ../datarepo-helm-definitions/dev/sh/datarepo-api.yaml
```

Comment:
^ This is the line I changed so that I didn't have to actually commit the above changes to my helm definitions.